saleor/checkout/utils.py

```diff
@@ -791,7 +792,12 @@ def _get_products_voucher_discount(checkout, voucher):
             "Voucher not applicable", "This offer is only valid for selected items."
         )
         raise NotApplicable(msg)
-    return get_products_voucher_discount(voucher, prices)
+    return get_products_voucher_discount(
+        voucher,
+        prices,
+        calculate_checkout_subtotal(checkout, discounts).gross,
```

Comment:
Minor stylistic issue - if you moved this to a variable, we would end up with two lines instead of six:
```
subtotal = calculate_checkout_subtotal(checkout, discounts).gross
return get_products_voucher_discount(voucher, prices, subtotal, checkout.quantity)
```